src/utils/make-reg-exp.js

```diff
@@ -9,7 +9,7 @@ export function parseRegExpString (regExp) {
     const parsedRegExpWithFlags = regExp.match(SPLIT_INPUT_AND_FLAGS_REG_EXP);
 
     if (parsedRegExpWithFlags)
-        return makeRegExp(parsedRegExpWithFlags[1], parsedRegExpWithFlags[2]);
+        return RegExp(parsedRegExpWithFlags[1], parsedRegExpWithFlags[2]);
```

Comment:
I guess you should remove outer quotes for the string `SkipJSOptions` options in the `src/utils/get-options/skip-js-errors.ts` file. Use the `src/utils/get-options/compiler.ts` file as an example.